<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dManualLoadMSS</class>
 <widget class="QDialog" name="dManualLoadMSS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Загрузка каналов МСС</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>29</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Номер блока</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBox">
       <property name="maximum">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Номер канала</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBox_2">
       <property name="maximum">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pbLoadMSS">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>251</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Включение/отключение блока МСС</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Номер блока:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbNumMSSfOnOff">
         <property name="maximum">
          <number>4</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="cbChannal1">
       <property name="text">
        <string>Канал 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbChannal2">
       <property name="text">
        <string>Канал 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbChannal3">
       <property name="text">
        <string>Канал 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbChannal4">
       <property name="text">
        <string>Канал 4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbOnOffMSS">
       <property name="text">
        <string>Выполнить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>481</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
